cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,30 @@
+class PaymentStepPage {
+    private paymentT: string;
+    private paymentC: string;
+    private confirmationO: string;
+    // private confirmationOrder: string;
+
+
+    constructor() {
+        this.paymentT = "a.bankwire";
+        this.paymentC = ".cart_navigation span";
+        this.confirmationO = "#center_column > div > p > strong";
+    }
+
+    public PaymentType(): void {
```

Comment:
It is a good practice to use verbs in infinitive mode on methods. You can use here SelectPayment or SelectBankwire as method name

Reply:
Changed to SelectPayment